### DISCUSSION TITLE
Update SvgText Content dynamically

### DISCUSSION THREAD

**Discussion by PraveenSuryawanshi-Dev** [vvvv/SVG#999] [Q&A]
#### Description

 var svgDoc  = SvgDocument.Open<SvgDocument>(stream);
 var bitmap0 = svgDoc .Draw(rasterWidth, rasterHeight);
 
svgDoc .GetElementById("#01").Children[1].Children[0].Content = "0";
 var bitmap1 = svgDoc .Draw(rasterWidth, rasterHeight);
 
svgDoc .GetElementById("#01").Children[1].Children[0].Content = "1";
 var bitmap2 = svgDoc .Draw(rasterWidth, rasterHeight);
 
 The pseudo example code doesn't update the modified content. Is it possible to modify the content without parsing the SVG from string again.

modifying the string format and reparsing to get the SvgDocument will get a latency so wanted to modify the part of the element dynamically and render again. 

Can you help me solve this problem? Please let us know if you have a reference to work or working code snippets. 
#### Used Versions

version : v4.0.30319

**Comment by H1Gdev** [comment]
@PraveenSuryawanshi-Dev 

Please use [Text property](https://github.com/svg-net/SVG/blob/v3.4.3/Source/Text/SvgTextBase.cs#L43-L53) for updating text.
